honeybee_vtk/model.py

```diff
@@ -187,6 +198,38 @@ def _load_cameras(self, model: HBModel) -> None:
                 'views not found in HBJSON.'
             )
 
+    def _load_data(self, config: dict) -> None:
+        """Mount data on model from config json.
+
+        This method is only used in cli.
+
+        Args:
+            config: A dictionary returned by check_data_config
```

Comment:
Why would you need check_data_config? Don't we use Pydantic to define the schema for config files?

Reply:
Why would I not check;

1. if it's a valid file path
2.  If it's a valid json
3.  Validate the file using schema

This comment is very confusing to me. May be you don't like the name of the method. I can rename it to get_config.

Reply:
Complied.